Makefile

```diff
@@ -102,7 +102,8 @@ manifests: $(CONTROLLER_GEN) $(YQ) $(YAMLFMT) ## Generate WebhookConfiguration,
 	$(CONTROLLER_GEN) crd paths="./apis/telemetry/v1alpha1" output:crd:artifacts:config=config/crd/bases
 	$(YQ) eval 'del(.. | select(has("otlp")).otlp)' -i ./config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml
 	$(YQ) eval 'del(.. | select(has("x-kubernetes-validations"))."x-kubernetes-validations"[] | select(.rule|contains("otlp")) )' -i ./config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml
-
+	## Remove empty arrays from logpipeline crd that can be caused by previous yq manipulations
+	$(YQ) eval 'del(.. | select(tag == "!!seq" and length == 0))' -i ./config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml
```

Comment:
that will remove all empty arrays...
why don't you just check the previously modified "x-kubernetes-validations" field?